src/v2i-hub/PedestrianPlugin/src/FLIRWebSockAsyncClnSession.cpp

```diff
@@ -6,10 +6,44 @@ using namespace boost::property_tree;
 
 namespace PedestrianPlugin
 {
+    int 
+    FLIRWebSockAsyncClnSession::calculateMinuteOfYear(int year, int month, int day, int hour, int minute, int second) {
```

Comment:
NIT: why is the method declaration on a separate line from the return type. This looks weird.

Reply:
Was following the format that was made by someone else. I don't like it either. I'll change all the declarations

Reply:
Was following the format that was made by someone else. I don't like it either. I'll change all the declarations